lerna.json

```diff
@@ -5,7 +5,18 @@
     ],
     "command": {
         "version": {
-            "message": "chore(release): Publish"
+            "message": "chore(release): Publish",
+            "allowBranch": [
```

Comment:
This will bring a bit of safety around releasing only in `main` or `next` branches.